docs/zkapps/testing-zkapps-lightnet.mdx

```diff
@@ -0,0 +1,346 @@
+---
```

Comment:
Make sure it 100% corresponds to what we currently have for https://docs.minaprotocol.com/zkapps/writing-a-zkapp/introduction-to-zkapps/testing-zkapps-lightnet
(including content itself + correct belonging to the docs structure)
Document is outdated and most likely it no longer belongs to this docs path.

Reply:
again a duplicate not displayed - as per sidebar.js - I've removed it.